### ISSUE TITLE
validationRegex issue

### ISSUE THREAD

**Issue by eddiemoore** [olahol/react-tagsinput#63]
With the latest update, i'm getting this issue:
```
Warning: Failed propType: TagsInput: prop type `validationRegex` is invalid; it must be a function, usually from React.PropTypes.
```
I looked at the code and it has a default. What am I missing here?

**Comment by olahol** [comment]
Looks like the prop type "regexp" did not exist in React 0.14. fixed in 3.4.1.

**Comment by eddiemoore** [comment]
:+1: Thanks
